src/django/api/migrations/0152_add_sector_group_table.py

```diff
@@ -0,0 +1,74 @@
+from django.db import migrations, models
+
+
+def populate_sector_groups(apps, schema_editor):
+    SectorGroup = apps.get_model('api', 'SectorGroup')
+    groups_names = [
+        "Apparel, Footwear, Textiles",
+        "Agriculture, Food & Beverage",
+        "Technology",
+        "Automotive",
+        "Electronics",
+        "Energy",
+        "Utilities",
+        "Mining",
+        "Construction",
+        "Cosmetics",
+        "Durable Goods",
+        "Transportation",
+        "Aerospace",
+        "Forestry",
+        "Press",
+        "Health",
+        "Entertainment",
+        "Accommodation",
+        "Governmental",
+        "Finance",
+        "General Merchandise",
+        "Storage",
+        "Sporting Goods",
+        "Education, Research, Services",
+        "Other",
+    ]
+
+    for name in groups_names:
+        SectorGroup.objects.create(name=name)
+
+
+def revert_populate_sector_groups(apps, schema_editor):
+    SectorGroup = apps.get_model('api', 'SectorGroup')
```

Comment:
I'm not sure that this revert is required as on revert action it should just delete model/table **SectorGroup**.

Reply:
Replaced it to `migrations.RunPython.noop`.